src/content/configuration/optimization.md

```diff
@@ -59,3 +60,49 @@ module.exports = {
 `object`
 
 By default webpack v4+ provides new common chunks strategies out of the box for dynamically imported modules. See available options for configuring this behavior in the [SplitChunksPlugin](/plugins/split-chunks-plugin/) page.
+
+## `optimization.runtimeChunk`
+
+`object` `string` `boolean`
+
+Setting `optimization.runtimeChunk` to `true` adds an additonal chunk to each entrypoint containing only the runtime.
+It is possible to use preset mode of the plugin by providing a string value:
+
+- `single`: creates a runtime file to be shared for all generated chunks.
+- `multiple`: creates multiple runtime files for common chunks.
+
+By setting `optimization.runtimeChunk` to `object` it is only possible to provide the `name` property which stands for the name or name factory for the runtime chunks.
+
+__webpack.config.js__
+
+
+```js
+module.exports = {
+  //...
+  optimization: {
+    runtimeChunk: {
+      name: entrypoint => `runtimechunk~${entrypoint.name}`
+    }
+  }
+}
+```
+
+## `optimization.noEmitOnErrors`
+
+`boolean`
+
+Use the `optimization.noEmitOnErrors` to skip the emitting phase whenever there are errors while compiling. This ensures that no erroring assets are emitted. The `emitted` flag in the stats is `false` for all assets.
+
+__webpack.config.js__
+
+
+```js
+module.exports = {
+  //...
+  optimization: {
+    noEmitOnErrors: true
+  }
+}
+```
+
+W> If you are using the [CLI](/api/cli/), the webpack process will not exit with an error code by enabling this plugin. If you want webpack to "fail" when using the CLI, please check out the [`bail` option](/api/cli/#advanced-options).
```

Comment:
_by enabling_ should be _while this plugin is enabled.

Reply:
true, i will also update this to be `while using webpack CLI`